collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -299,7 +299,6 @@ protected void updateAdapter() {
             adapter.notifyDataSetChanged();
```

Comment:
Shouldn't we remove this else block instead?

Reply:
Why? notifyDataSetChanged() should be called only then - if the adapter is updated, it doesn't make sense to do that when a new object is created.

Reply:
You're absolutely correct, my bad